src/component/academic/nilai/nilai.jsx

```
@@ -1,14 +1,19 @@
-      <h1>Nilai</h1>
-      <div className="nilai-content">
-        {/* Consider adding actual grade display table or cards here */}
-        <p>Grade information will be displayed here</p>
-        </div>
-    </div>
-  );
-};
-export default Nilai;
+import Sidebar from "../../sidebar/sidebar.jsx";
+
+const dataPengetahuan = [
```

ESLint

[error] 'dataPengetahuan' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)